lib/controllers/register.js

```diff
@@ -89,6 +89,20 @@ function applyDefaultAccountFields(config,req) {
     req.body.surname = 'Anonymous';
   }
 }
+/**
```

Comment:
After looking at this again, the fallback error message is specific to this registration behavior.  I don't see a need to break this out to a file (yet).